spec/datadog/opentracer/text_map_propagator_spec.rb

```diff
@@ -51,13 +51,13 @@
         end
 
         expect(carrier).to have_received(:[]=)
-          .with(Datadog::OpenTracer::DistributedHeaders::HTTP_HEADER_TRACE_ID, trace_id)
+          .with('x-datadog-trace-id', trace_id)
         expect(carrier).to have_received(:[]=)
-          .with(Datadog::OpenTracer::DistributedHeaders::HTTP_HEADER_PARENT_ID, span_id)
+          .with('x-datadog-parent-id', span_id)
         expect(carrier).to have_received(:[]=)
-          .with(Datadog::OpenTracer::DistributedHeaders::HTTP_HEADER_SAMPLING_PRIORITY, sampling_priority)
+          .with('x-datadog-sampling-priority', sampling_priority)
         expect(carrier).to have_received(:[]=)
-          .with(Datadog::OpenTracer::DistributedHeaders::HTTP_HEADER_ORIGIN, origin)
+          .with(Datadog::OpenTracer::DistributedHeaders::ORIGIN_KEY, origin)
```

Comment:
I... don't think `ORIGIN_KEY` is in this class anymore? (also applies to a few other opentracer specs as well)

Reply:
Forgot to inline this one 😓 .
Done!